README

```diff
@@ -0,0 +1,19 @@
+hc-ansible-deployment is end-to-end HC deployment playbook which is using both gluster-ansible role and ovirt-ansible-hosted-engine-setup role.
+1. There is one inventory file which is gluster_inventory.yml/gluster_inventory_vdo.yml has gluster related configurations
```

Comment:
why do we need 2 inventory files? Can the vdo variables be captured in gluster_inventory as well (commented out if not needed)?

Reply:
Included vdo configs in gluster_inventory and removed gluster_inventory_vdo.yml